java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -65,6 +65,18 @@
         Kind(int nativeId) {this.nativeId = nativeId;}
     }
 
+    /*
+     * This is analogous to the native 'null_policy'.
+     */
+    public enum NullPolicy {
```

Comment:
The only problem with this is that count, nth, and nunique done include APIs that take this enum too.  I just want it to be consistent. We cannot remove the existing boolean APIs so we would need to add new ones and deprecate the existing ones.

Reply:
Hmm. Deprecation would indeed be the end-game. 
@revans2, would you recommend that we hold off on the `NullPolicy` change for just this API, and do a separate PR to introduce `NullPolicy` and deprecate the bool APIs?

Reply:
I am fine either way. It is not a big change, I just want the APIs to be consistent.

Reply:
Thanks, @revans2.

@firestarman, I apologize for the churn. Maybe we can introduce `NullPolicy` in a separate PR, and change `count`, `nth`, `nunique`, and `collect` together to use it.

Reply:
OK. 
It is likely to require changes in plugin where using these aggregations after involving the `NullPolicy `, so we need to change these carefully.

Reply:
@mythrocks  @revans2 Updated, could you help review again ?